File: java/src/SimpleArraySum.java
import java.util.Arrays;

public class SimpleArraySum {

    public static int simpleArray(int[] array){
        int accumulator = 0;
        int lengthOfArray = array.length;
        int start = 0;
        for (int i = start; i < lengthOfArray; i++){
            accumulator += array[i];
        }
        return accumulator;
    }

    public static void main(String[] args) {
        int[] array = {1, 2, 3, 4, 10, 11};
        int total = simpleArray(array);
        System.out.println("Result-> " + total);
    }
}

File: javascript/simple_array_sum.js
function simpleArraySum(arr) {
    const total = arr.reduce((accumulator, currentValue) => {
        return accumulator + currentValue;
    }, 0)
    return total;
}

function display() {
    const array = [1, 2, 3, 4, 10, 11];
    const result = simpleArraySum(array);
    console.log("Result-> " + result);
}
display();
